swarm/network/pss.go

```diff
@@ -371,7 +403,9 @@ func (prw PssReadWriter) WriteMsg(msg p2p.Msg) error {
 
 	pmsg, _ := makeMsg(msg.Code, ifc)
 
-	return prw.Pss.Send(to, *prw.topic, pmsg)
+	prw.Pss.Send(to, *prw.topic, pmsg)
```

Comment:
It might have been related to testing problems. I'll change it back, I can't remember if there was a better reason.